knife/lib/chef/knife/core/gem_glob_loader.rb

```diff
@@ -45,22 +45,13 @@ def gem_and_builtin_subcommands
           find_subcommands_via_dirglob
         end
 
-        def find_subcommands_via_dirglob
-          # The "require paths" of the core knife subcommands bundled with chef
```

Comment:
I removed this because the same method is defined in  SubcommandLoader ; the only difference is that this one does not exclude 'version.rb' ; and it references KNIFE_ROOT instead of CHEF_ROOT.